### PR TITLE
fix levit timm conversion file

### PR DESCRIPTION
**Pull request by Bearnardd**
# What does this PR do?
Fixes conversion file `convert_levit_timm_to_pytorch.py` for levit

Fixes # (issue)
https://github.com/huggingface/transformers/issues/20937

## Who can review?
@NielsRogge

### CODE REVIEW

**Review by CloudInnovator — APPROVED** [review]
Changes follow convention and all tests passed.

File: src/transformers/models/levit/convert_levit_timm_to_pytorch.py [hunk 1]
```diff
@@ -167,13 +167,14 @@ def convert_weights_and_push(save_directory: Path, model_name: str = None, push_
...
-        default=True,
-        type=bool,
-        required=False,
-        help="If True, push model and feature extractor to the hub.",
+        "--no-push_to_hub",
+        dest="push_to_hub",
+        action="store_false",
+        help="Do not push model and feature extractor to the hub",
     )
+    parser.set_defaults(push_to_hub=True)
```
Comment (NielsRogge) [inline src/transformers/models/levit/convert_levit_timm_to_pytorch.py:177]: (nit) I'm not using this in any conversion script

**Review by NielsRogge — APPROVED** [review]
Thanks for fixing!

File: src/transformers/models/levit/convert_levit_timm_to_pytorch.py (hunk 1 above)
Comment (Bearnardd) [reply src/transformers/models/levit/convert_levit_timm_to_pytorch.py]: @NielsRogge  removed.

**Comment by sgugger** [comment]
Thanks again for your contribution!

**Pull request merged**